addons/mail/static/src/components/attachment/attachment.js

```diff
@@ -130,7 +135,7 @@ class Attachment extends Component {
      * @param {MouseEvent} ev
      */
     _onClickUnlink(ev) {
-        ev.stopPropagation();
+        markEventHandled(ev, 'Attachment.actionButton');
```

Comment:
Would clarify exactly action that is being handled here: `Attachment.clickUnlink`.